ServerModules/package-lock.json

```diff
@@ -1,39 +1,8 @@
 {
   "name": "pirate-panic-server",
-  "version": "1.0.0",
```

Comment:
Are you still able to develop locally with type definitions after removing this?

Reply:
Yes but the reason it changed so much is because I was running npm v6 instead of v7, so it looks like it reverted to the older lockfile version 2. I'll fix that, one sec.